.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
- `required`
- **Well done so far👏👏**

- [ ] I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate 😎.

- [ ] I also strongly suggest that in order to sound polite you need to be using words like `please`, `kindly`, `I suggest`, and `I highly recommend` ...etc, and many other words you find properly.

- [ ] Please explain the **why** you are asking for that change by providing examples and best practices approach ✔️  so the student can learn something 👨‍🏫😄 as well and don't feel the change was unfair 👌.

Reply:
## _Thank you for your review!_  🫡
### _I've implemented the following changes to my review comments_
-  ✔️ _Requested the changes on a more kindly way!_
- ✔️ _Explained why I was asking those changes_
- ✔️ _Added emojis to personalize the comments 🥸_